master/getting-started/kubernetes/installation/integration.md

```diff
@@ -80,7 +80,7 @@ Environment=ETCD_ENDPOINTS=http://<ETCD_IP>:<ETCD_PORT>
 PermissionsStartOnly=true
 ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node \
   -e ETCD_ENDPOINTS=${ETCD_ENDPOINTS} \
-  -e HOSTNAME=${HOSTNAME} \ 
+  -e HOSTNAME=${HOSTNAME} \
```

Comment:
Would prefer to switch over to using NODENAME for the container environments now (both are still supported though)